docs/usage.md

```diff
@@ -1,63 +1,62 @@
+<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
 
-```bash
---input '[path to samplesheet file]'
-```
+## Prerequisites
 
-### Multiple runs of the same sample
+1. Install Nextflow (>=22.10.1) using the instructions [here.](https://nextflow.io/docs/latest/getstarted.html#installation)
+2. Install one of the following technologies for full pipeline reproducibility: Docker, Singularity, Podman, Shifter or Charliecloud.
 
-The `sample` identifiers have to be the same when you have re-sequenced the same sample more than once e.g. to increase sequencing depth. The pipeline will concatenate the raw reads before performing any downstream analysis. Below is an example for the same sample sequenced across 3 lanes:
+## Samplesheet input
 
-```csv title="samplesheet.csv"
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L003_R1_001.fastq.gz,AEG588A1_S1_L003_R2_001.fastq.gz
-CONTROL_REP1,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz
-```
+You will need to create a samplesheet in csv format with information about the samples you would like to analyse before running the pipeline. It has to be a comma-separated file and a header row as shown in the examples below.
 
-### Full samplesheet
+This samplesheet is then specified on the command line as follows:
 
-The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first 3 columns to match those defined in the table below.
+```bash
+--input '[path to samplesheet file]'
+```
 
-A final samplesheet file consisting of both single- and paired-end data may look something like the one below. This is for 6 samples, where `TREATMENT_REP3` has been sequenced twice.
+genomic-medicine-sweden/meta-val will requires the information given bellow.
+
+| Column              | Description                                                                                                                               |
+| ------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| sample              | Unique sample name [required].                                                                                                            |
+| run_accession       | Run ID or name unique for each (pairs of) file(s). Can also supply sample name again here, if only a single run was generated [required]. |
+| instrument_platform | Sequencing platform reads generated on, selected from the EBI ENA controlled vocabulary [required].                                       |
```

Comment:
What do you mean with EBI ENA controlled vocabulary?

Reply:
I copied it from `taxprofiler` https://github.com/nf-core/taxprofiler/blob/f1e6153e4234c899c5db50e58638fddb58b9f219/docs/usage.md?plain=1#L85